### PR TITLE
New commute data

### PR DESCRIPTION
**Pull request by mem48**
@Robinlovelace I've created the new flow data and a PR to add to targets.

A few things to check

1. I'm having problems again with the jittering - keep hanging or crashing
2. You need https://github.com/nptscot/inputdata/blob/main/DataZones.Rds in the inputdata folder
3. Now we are using secure data do we need to check for any public release of the OD/routes data? Need same approach as already being used for schools data. I see current releases e.g. https://github.com/nptscot/npt/releases/tag/v2023-07-19-21-24-37_commit_58dd9879df5ea5b70355b36f2e2a47edeb3202e9 are publishing the routes.

### CODE REVIEW

**Comment by Robinlovelace** [comment]
> I'm having problems again with the jittering - keep hanging or crashing

I think this is the same as https://github.com/dabreegster/odjitter/issues/49

Try setting the deduplicate argument to FALSE.

**Comment by Robinlovelace** [comment]
> 1. I'm having problems again with the jittering - keep hanging or crashing
> 

See response above, I suggest not deduplicating for now.

>     2. You need https://github.com/nptscot/inputdata/blob/main/DataZones.Rds in the inputdata folder

Sounds good and great work on creating that.

> 
>     3. Now we are using secure data do we need to check for any public release of the OD/routes data? Need same approach as already being used for schools data. I see current releases e.g. https://github.com/nptscot/npt/releases/tag/v2023-07-19-21-24-37_commit_58dd9879df5ea5b70355b36f2e2a47edeb3202e9 are publishing the routes.

Yes. @joeytalbot please don't add releases to this repo but to the https://github.com/nptscot/outputdata/releases repo

**Comment by mem48** [comment]
this now works for a mini-test build. had to change some uptake code due to different mode splits

**Comment by mem48** [comment]
@Robinlovelace I think this can now be merged, please review. Once this is in the main branch I can start on the zones

File: R/uptake.R [hunk 1]
```diff
@@ -16,11 +16,11 @@ get_scenario_go_dutch = function(routes, purpose = "work") {
         bicycle_go_dutch = pcycle_go_dutch * all,
         bicycle_increase = bicycle_go_dutch - bicycle,
         car_driver_go_dutch = car_driver - (bicycle_increase * car_driver / all),
-        car_passenger_go_dutch = car_passenger - (bicycle_increase * car_passenger / all),
+        #car_passenger_go_dutch = car_passenger - (bicycle_increase * car_passenger / all),
```
Comment (Robinlovelace) [inline R/uptake.R:19]: We don't need car for now. How hard would it be to add @mem48 ?

File: _targets.R [hunk 2]
```diff
@@ -77,11 +77,22 @@ list(
   # }),
   tar_target(od_data, {
     min_flow = parameters$min_flow # Set to 1 for full build, set to high value (e.g. 400) for tests
-    desire_lines_raw = readRDS("inputdata/desire_lines_scotland.Rds")
+    
+    #desire_lines_raw = readRDS("inputdata/desire_lines_scotland.Rds")
+    path_teams = Sys.getenv("NPT_TEAMS_PATH")
+    if(nchar(path_teams) == 0){
```
Comment (Robinlovelace) [inline _targets.R:83]: Good defensive programming.

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
> Now we are using secure data do we need to check for any public release of the OD/routes data? Need same approach as already being used for schools data. I see current releases e.g. https://github.com/nptscot/npt/releases/tag/v2023-07-19-21-24-37_commit_58dd9879df5ea5b70355b36f2e2a47edeb3202e9 are publishing the routes.

Heads-up @mem48 and @joeytalbot I've now deleted that as it should be in outputdata.

File: R/uptake.R (hunk 1 above)
Comment (mem48) [reply R/uptake.R:19]: In Scotland there is no driver/passenger  split but we do have car as a single category

**Pull request merged**